gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/BaseIcebergCatalog.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.gobblin.data.management.copy.iceberg;
 
+import java.io.IOException;
```

Comment:
likely to be a lint error

(FYI in other files too)

Reply:
yeah... removed the unused imports in my latest commit!